src/components/ViewerSettings.vue

```diff
@@ -145,6 +153,15 @@ export default class ViewerSettings extends Vue {
     this.store.setScaleAnnotationsWithZoom(value);
   }
 
+  get annotationOpacity() {
+    return this.store.annotationOpacity;
+  }
+
+  set annotationOpacity(value: number) {
+    const opacity = typeof value === "string" ? parseFloat(value) : value;
+    this.store.setAnnotationOpacity(opacity);
+  }
```

Comment:
I'm not sure how many input events the slider sends when you move it, but since each time you mutate the store, I'm afraid this will slow down the application.  
My recommendation would be to add a `@Debounce(...)` on that function.  
We have examples of this in the codebase.

Reply:
Updated, is this right?